k9mail/src/main/java/security/planck/ui/passphrase/PassphrasePresenter.kt

```diff
@@ -82,7 +85,17 @@ class PassphrasePresenter @Inject constructor(
     }
 
     fun validateInput(passphrase: String) {
-        view.enableActionConfirmation(passphrase.isNotEmpty())
+        val isValidPassPhrase = passphrase.isValidPassphrase()
+        view.enableActionConfirmation(isValidPassPhrase)
+        if (isValidPassPhrase) {
+            view.hidePassphraseError()
+        } else {
+            view.showPassphraseError()
```

Comment:
if there is an error, should we say what are the requirements to the user? Otherwise he would not know and maybe also this is to be made configurable

Reply:
Yes, the implementation in the view is displaying the requeriments to the user.
<img width="432" alt="image" src="https://github.com/plancksecurity/android-planckForAndroid/assets/151338104/0a3fa56b-5b6b-4a3e-a713-fef06a00ddfb">

Reply:
perfect, if it is not configurable then it is great as it is. If we get a story asking to configure it we can always adjust.